iDiff/cmd/iDiff.go

```diff
@@ -31,12 +31,40 @@ var iDiffCmd = &cobra.Command{
 				fmt.Println(err)
 			}
 			fmt.Println(diff)
+		} else if args[2] == "apt" {
+			diff, err := aptDiff(args[0], args[1])
+			if err != nil {
+				fmt.Println(err)
+			}
+			fmt.Println(diff)
 		} else {
 			fmt.Println("Unknown differ")
 		}
 	},
 }
 
+func aptDiff(img1, img2 string) (string, error) {
+	jsonPath1, dirPath1, err := utils.ImageToDir(img1)
+	if err != nil {
+		return "", err
+	}
+	jsonPath2, dirPath2, err := utils.ImageToDir(img2)
+	if err != nil {
+		return "", err
+	}
+	diff, err := differs.AptDiff(dirPath1, dirPath2)
+	if err != nil {
+		return "", err
+	}
+
+	defer os.RemoveAll(dirPath1)
```

Comment:
RemoveAll and Remove throw error that you might want to catch.  As these methods are at the end, you don't need to defer calls and that way you can examine the errors.